### PR TITLE
Adds the admin only Ratvarian escape shuttle and some minor fixes to my modular bars.

### PR DESCRIPTION
**Pull request by wraith-54321**
# About The Pull Request

Adds a new admin only escape shuttle and fixes some minor issues with some of the modular bars I added.

## Why It's Good For The Game

Clock cult shuttle to go along with the blood cult shuttle, and fixes are generally good.

## Testing Photographs and Procedure

<details>

<summary>Screenshots&Videos</summary>

The escape shuttle

Screenshot (183)

New version of shuttle.

Screenshot (184)

</details>

## Changelog

:cl:
add: New admin only Ratvarian escape shuttle
fix: Added grills to the clockwork bar windows
fix: Removed duplicate chairs in the magical bar
/:cl:

### CODE REVIEW

**Review by nednaZ — CHANGES_REQUESTED** [review]
The shuttle section should have seats.
The engines are very weird and uneven for both sides.
Needs some form of brig section to make it fully usable.
Get rid of two of the clockwork warriors and the strangely placed pasta.

File: monkestation/code/datums/shuttles.dm [hunk 1]
```diff
@@ -21,3 +21,13 @@
 	description = "A heavily retrofitted Box-Model exfiltration vessel, complete with private rooms for every non-civilian \
 	department. Rated maximum safe occupancy: 20. Exceeding this limit may result in the witholding of your health benefeits."
 	admin_notes = "Designed for lowpop rounds and will be exceedingly chaotic outside of that setting."
+
+/datum/map_template/shuttle/emergency/brass
+	prefix = "monkestation/_maps/shuttles/"
+	suffix = "brass"
+	name = "Ratvarian Emergency Shuttle"
+	credit_cost = 10000
```
[the remark this answers is not in the document]
Comment (wraith-54321) [reply monkestation/code/datums/shuttles.dm:29]: The pasta is a reference to the ark which gives pasta if you use a fork on it, might take it down to just one. The brig is the room with the observation consoles.

[the remark this answers is not in the document]
Comment (wraith-54321) [reply monkestation/code/datums/shuttles.dm:29]: The cogscarabs are shells due to the fact that spawning in the mob itself runtimes for some reason, however they cant do much besides build unless Reebe is loaded

[the remark this answers is not in the document]
Comment (wraith-54321) [reply monkestation/code/datums/shuttles.dm]: I added a screenshot of the new version of the shuttle to the screenshots section.

**Comment by wraith-54321** [comment]
The cogscarabs need them to be able to unlock defense buildings as I dont know any other way to give them some cogs and I dont know what you mean about them not having time to add gears.

**Comment by nednaZ** [comment]
The balloon that gives ghost roles to the cogscarabs only activates about 30 seconds before the shuttle arrives.
The scarabs have less than ten seconds before shuttle arrival, so there is zero time to build defenses or get cogs into the APCs.

**Comment by wraith-54321** [comment]
As I said here

> The cogscarabs are shells due to the fact that spawning in the mob itself runtimes for some reason

The cogscarabs are shells and can be entered by ghosts as a role, part of the gimmick is they can build up defenses before the shuttle arrives like on reebe, the shuttle has a wall of servant blockers around the entire area they can walk around in so they cant get out.

**Review by nednaZ — APPROVED** [review]
The APC justification makes sense, even if it's a bit weird in appearances.
Worth trying this shuttle out

**Pull request merged**